src/sample/lecDashController.java

```diff
@@ -114,22 +115,23 @@ public void saveInfo() {
         Message.setText("Please enter a NRIC number with 12 digits!");
       } else{ */
 
-        Message.setText("Save successful!");
-        try{
-          File lecinfoFile = new File("data/Lecturer Dashboard/"+ id +".txt");
-          lecinfoFile.createNewFile();
-          BufferedWriter writer=new BufferedWriter(new FileWriter("data/Lecturer Dashboard/"+ id +".txt"));
-          writer.write(name.getText());
-          writer.write("\n" + nric.getText());
-          writer.write("\n" + id);
-          writer.write("\n" + emp_status.getSelectionModel().getSelectedItem());
-          writer.write("\n" + position.getSelectionModel().getSelectedItem());
-          writer.write("\n" + school.getSelectionModel().getSelectedItem());
-          writer.write("\n" + campus.getSelectionModel().getSelectedItem() + "\n");
-          writer.close();
-        } catch(IOException e) {
-          System.out.println("An error occured.");
-        }
+        
+    try{
+      //File lecinfoFile = new File("data/Lecturer Dashboard/"+ id +".txt");
```

Comment:
can delete unused comments

Reply:
okay, later I delete, I would like to make sure there are really no more bug first